src/goto-symex/goto_symex.h

```diff
@@ -171,8 +171,6 @@ class goto_symext
   const bool allow_pointer_unsoundness;
 
 public:
-  // these bypass the target maps
-  virtual void symex_step_goto(statet &, bool taken);
```

Comment:
Maybe the commit message could be made to include the names of functions being removed? These have likely been used elsewhere (Satabs comes to mind), and if we ever want to resurrect them, the git history is vital.